src/Structures/VoiceState.ts

```diff
@@ -1,3 +1,68 @@
-export class VoiceState {
-    
-}
\ No newline at end of file
+import { DiscordVoiceState } from 'discordeno/types';
+import { Base } from '../Base.js';
+
+export class VoiceState extends Base {
+    channelID!: string | null;
+    deaf: boolean;
```

Comment:
All of these booleans should get merged into a toggle handler with bitfields like we do in discordeno

Reply:
With the get methods like these?

![image](https://user-images.githubusercontent.com/38146668/196043927-3eedee9c-90f8-4e75-955b-2f07be90743f.png)